src/node_options.h

```diff
@@ -231,6 +231,7 @@ class PerProcessOptions : public Options {
 
 #ifdef NODE_REPORT
   std::vector<std::string> cmdline;
+  bool report_on_fatalerror = false;
```

Comment:
Can you also remove the `report_on_fatalerror` option from the per-env option list?

Reply:
Is there any other case where we might need it. I left it as I was not clear on that.

Reply:
@shobhitchittora I’m not sure what you mean by that – you’ve moved the option to a wider scope, so the original field is now useless and can be removed.

(You do need to update the references in `node_report_module.cc` as well in order for that to work.)

Reply:
Okay! Got it.